tests/unittests/tensorsTest.cpp

```diff
@@ -652,6 +652,50 @@ TEST(Tensor, insertWithCountAndAxis) {
   }
 }
 
+/// Verify that tensors that are quantized begin zero'd to their type's offset
+/// and are reset back to that offset.
+TEST(Tensor, zeroQuanitizedTensor) {
```

Comment:
typo in `zeroQuanitizedTensor`

Reply:
can you also add a test case where you set values to the tensor to be equal to the offset and isZero returns true?